proxybroker/utils.py

```diff
@@ -58,6 +58,8 @@ def get_all_ip(page):
 
 def get_status_code(resp, start=9, stop=12):
     try:
+        if not isinstance(resp, (bytes, str)):
```

Comment:
From the viewpoint to check the external argument is acceptable, but the resp in this module should be all bytes object, and what I would like to know is there any edge case happened?